litmus-portal/frontend/src/pages/Workflows/index.tsx

```diff
@@ -37,7 +45,7 @@ const Workflows = () => {
           <div className={classes.scheduleBtn}>
             <ButtonFilled
               isPrimary={false}
-              handleClick={() => history.push('/create-workflow')}
+              handleClick={handleScheduleWorkflow}
             >
               <>Schedule a workflow</>
```

Comment:
Please will you put this as a part of the translation!

Reply:
Yes Sure

Reply:
Resolved :+1: